tokens.xml

```diff
@@ -6325,6 +6438,21 @@ Equip {1}</text>
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Rogue Token</name>
+            <text></text>
+            <prop>
+                <colors>B</colors>
+                <type>Token Creature — Rogue</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>2/2</pt>
```

Comment:
what's our policy on tapped tokens? do we add cipt?

Reply:
currently Ragavan is the one token using cipt, I think this is more suited to a cockatrice change instead of setting the token to cipt  https://github.com/Cockatrice/Cockatrice/issues/4573